test_runner/regress/test_threshold_based_eviction.py

```diff
@@ -52,8 +52,8 @@ def test_threshold_based_eviction(
         "kind": "NoEviction"
     }
 
-    eviction_threshold = 5
-    eviction_period = 1
+    eviction_threshold = 10
+    eviction_period = 2
```

Comment:
Should this use the `ATIME_RESOLUTION` variable introduce in `test_disk_usage_based_eviction` or it is by accident that this is also `2`?

Reply:
It's a coincidence: these are just arbitrarily 2x the old values